Gemfile

```diff
@@ -61,6 +62,7 @@ group :test do
   gem 'shoulda-matchers', '~> 3.1.1'
   gem 'simplecov', '~> 0.13.0', require: false
   gem 'action-cable-testing', '~> 0.3.2'
+  gem 'webmock', '~> 3.4', '>= 3.4.2'
```

Comment:
Great! Since you are adding webmock is a good practice to add the following line to your `rails_helper.rb`:
```
WebMock.disable_net_connect!(allow_localhost: true)
```

This will prevent to do external call during the tests